### PR TITLE
refactor: rename "AirstreamCampaign" to "MerkleStreamer"

### PR DESCRIPTION
**Pull request by PaulRBerg**
Implements https://github.com/sablier-labs/v2-periphery/discussions/183.

Note that I have kept referring to the total amount as "campaign total amount", and to the expiration as "campaign expiration".

### CODE REVIEW

File: test/utils/Precompiles.t.sol [hunk 1]
```diff
@@ -26,9 +26,9 @@ contract Precompiles_Test is Base_Test {
         }
     }
 
-    function test_DeployAirstreamCampaignFactory() external onlyTestOptimizedProfile {
-        address actualFactory = address(precompiles.deployAirstreamCampaignFactory());
-        address expectedFactory = address(deployPrecompiledAirstreamCampaignFactory());
+    function test_DeployMerkleStreamerFactory() external onlyTestOptimizedProfile {
```
Comment (andreivladbrg) [inline test/utils/Precompiles.t.sol:29]: Not alphabetically ordered

File: test/utils/Precompiles.t.sol [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { IAllowanceTransfer } from "@uniswap/permit2/interfaces/IAllowanceTransf
 import { DeployPermit2 } from "@uniswap/permit2-test/utils/DeployPermit2.sol";
 import { LibString } from "solady/utils/LibString.sol";
 
-import { ISablierV2AirstreamCampaignFactory } from "../../src/interfaces/ISablierV2AirstreamCampaignFactory.sol";
+import { ISablierV2MerkleStreamerFactory } from "../../src/interfaces/ISablierV2MerkleStreamerFactory.sol";
```
Comment (andreivladbrg) [inline test/utils/Precompiles.t.sol:8]: Not alphabetically ordered

File: test/utils/Precompiles.t.sol [hunk 3]
```diff
@@ -102,7 +102,7 @@ contract Precompiles_Test is Base_Test {
 
         vars.permit2 = IAllowanceTransfer(new DeployPermit2().run());
         (
-            vars.actualAirstreamFactory,
+            vars.actualMerkleStreamerFactory,
```
Comment (andreivladbrg) [inline test/utils/Precompiles.t.sol:105]: Not alphabetically ordered

File: test/integration/merkle-streamer/campaign-ll/claim/claim.tree [hunk 4]
```diff
@@ -16,5 +16,5 @@ claim.t.sol
       │     └── it should revert
       └── given the claim is included in the Merkle tree
          ├── it should mark the index as claimed
-         ├── it should create a LockupLinear airstream
+         ├── it should create a LockupLinear Merkle streamer
```
Comment (andreivladbrg) [inline test/integration/merkle-streamer/campaign-ll/claim/claim.tree:19]: Typo: stream

File: test/fork/assets/USDT.t.sol [hunk 5]
```diff
@@ -3,7 +3,7 @@ pragma solidity >=0.8.19 <0.9.0;
 
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
-import { AirstreamCampaignLL_Fork_Test } from "../airstream/CampaignLL.t.sol";
+import { MerkleStreamerLL_Fork_Test } from "../merkle-streamer/MerkleStreamerLL.t.sol";
```
Comment (andreivladbrg) [inline test/fork/assets/USDT.t.sol:6]: Not alphabetically ordered

File: test/fork/assets/USDC.t.sol [hunk 6]
```diff
@@ -3,7 +3,7 @@ pragma solidity >=0.8.19 <0.9.0;
 
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
-import { AirstreamCampaignLL_Fork_Test } from "../airstream/CampaignLL.t.sol";
+import { MerkleStreamerLL_Fork_Test } from "../merkle-streamer/MerkleStreamerLL.t.sol";
```
Comment (andreivladbrg) [inline test/fork/assets/USDC.t.sol:6]: Not alphabetically ordered

**Review by andreivladbrg — CHANGES_REQUESTED** [review]
Shouldn't we rename [this dir](https://github.com/sablier-labs/v2-periphery/tree/refactor/rename-airstreams/test/integration/merkle-streamer/campaign-ll) to `ll`

**Comment by PaulRBerg** [comment]
Thanks for the review.

I have accepted all of your proposed changes and merged the other PR.

**Comment by PaulRBerg** [comment]
Once the CI checks finish, we can squash and merge the PR.

**Pull request merged**